packages/unchained-client/src/dogecoin/index.ts

```diff
@@ -0,0 +1 @@
+export * as dogecoin from '../bitcoin'
```

Comment:
prefer export the generated/dogecoin (even though the same right now just to be more accurate)
and then can export parser and types from bitcoin for now until updated to utxo parser (similar to evm).

this as cast also seems incorrect resulting in `unchained-client.dogecoin.dogecoin`